src/aihwkit/inference/noise/pcm.py

```diff
@@ -78,6 +83,7 @@ def __init__(
         read_noise_scale: float = 1.0,
         drift_scale: float = 1.0,
         prog_coeff_g_max_reference: Optional[float] = None,
+        **kwargs: Any,
```

Comment:
Best not to use "kwargs" here. Why not simply add new explicit args? If this changes old checkpoints, maybe best to just write a new noise model class with custom drift model?